aw_watcher_afk/unix.py

```diff
@@ -37,17 +37,17 @@ def time_since_last_input(self) -> float:
 _last_input_unix = None
 
 
-def time_since_last_input():
+def seconds_since_last_input():
     global _last_input_unix
```

Comment:
I know that you didn't write this, but it'd be better to attach the LastInputUnix object to the function object.

```python
def seconds_since_last_input():
    if not hasattr(seconds_since_last_input, "last_input_unix"):
        seconds_since_last_input.last_input_unix = LastInputUnix()
    return seconds_since_last_input.last_input_unix.seconds_since_last_input()
```

This way there is no global.